src/microbe_stage/Microbe.cs

```diff
@@ -789,21 +789,55 @@ public void Divide()
         copyEntity.Compounds.ClearCompounds();
 
         var keys = new List<Compound>(Compounds.Compounds.Keys);
+        var reproductionCompounds = CalculateTotalCompounds();
 
-        // Split the compounds evenly between the two cells.
+        // Split the compounds between the two cells.
         foreach (var compound in keys)
         {
             var amount = Compounds.GetCompoundAmount(compound);
 
             if (amount > 0)
```

Comment:
The indentation level starts to be a bit high in this loop.
I think this if doesn't have an else. Would be nicer now to swap this around so that
``` 
if (amount <= 0)
    continue;
```